_tags

```diff
@@ -16,57 +16,55 @@ true: annot, bin_annot
 "_darcs": not_hygienic
 # Library sqlexpr
 "src/sqlexpr.cmxs": use_sqlexpr
-<src/*.ml{,i,y}>: pkg_csv
```

Comment:
Do you know if the `pkg_xxx` tags are deprecated and whether package(xxx) introduces a compatibility issue with older ocamlfind? (not that it'd matter a lot, but it'd be better to add a version constraint in `opam` if so)